enterprise_access/apps/bffs/serializers.py

```diff
@@ -0,0 +1,40 @@
+"""
+Serializers for bffs.
+"""
+from rest_framework import serializers
+
+
+class ErrorSerializer(serializers.Serializer):
```

Comment:
[suggestion] Consider having a base serializer for `developer_message`, `user_message`, and the `create`/`update` methods.